src/logger.ts

```diff
@@ -0,0 +1,6 @@
+import { createLogger, EncoderTypeEnum } from "@fwl/logging";
+
+export const logger = createLogger({
+  service: "connect-account",
```

Comment:
Should we name `TRACER_SERVICE_NAME` `SERVICE_NAME` and use it both for the tracer and the logs? 😇

Reply:
Smart move.

Reply:
Do you agree with
```ts
config.serviceName = process.env.SERVICE_NAME || "Connect.Account";
```

Reply:
yes